CMakeLists.txt

```diff
@@ -110,20 +110,20 @@ if(BUILD_SHARED_LIBS)
     message(STATUS "Building a shared library")
 endif()
 
-# note: CONFIG_TSAN is currently incompatible with the other sanitizers but we
+# note: QJS_ENABLE_TSAN is currently incompatible with the other sanitizers but we
 # don't explicitly check for that because who knows what the future will bring?
-# CONFIG_MSAN only works with clang at the time of writing; also not checked
+# QJS_ENABLE_MSAN only works with clang at the time of writing; also not checked
 # for the same reason
-xoption(BUILD_EXAMPLES "Build examples" OFF)
-xoption(BUILD_STATIC_QJS_EXE "Build a static qjs executable" OFF)
-xoption(BUILD_CLI_WITH_MIMALLOC "Build the qjs executable with mimalloc" OFF)
-xoption(BUILD_CLI_WITH_STATIC_MIMALLOC "Build the qjs executable with mimalloc (statically linked)" OFF)
-xoption(CONFIG_ASAN "Enable AddressSanitizer (ASan)" OFF)
-xoption(CONFIG_MSAN "Enable MemorySanitizer (MSan)" OFF)
-xoption(CONFIG_TSAN "Enable ThreadSanitizer (TSan)" OFF)
-xoption(CONFIG_UBSAN "Enable UndefinedBehaviorSanitizer (UBSan)" OFF)
-
-if(CONFIG_ASAN)
+xoption(QJS_BUILD_EXAMPLES "Build examples" OFF)
+xoption(QJS_BUILD_STATIC_EXE "Build a static qjs executable" OFF)
```

Comment:
Now that I see them side by side, I guess this would better be `QJS_BUILD_CLI_STATIC` ?

Reply:
Done 😄